inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -395,14 +398,27 @@ ModelPtr FrontEnd::runCommonPasses(ie::ICNNNetwork& network, const UnsupportedLa
             ngraph::op::GenericIE::DisableReshape noReshape(nGraphFunc);
 
             ngraph::pass::Manager manager;
+            const auto& env = CompileEnv::get();
+            if (!env.config.forcePureTensorIterator) {
```

Comment:
let's do not include this passes into CommonOptimization until we have not mechanism for transformations disabling.

Reply:
these changes were moved to a separate branch